src/components/activity/ActivitySpanForm.svelte

```diff
@@ -148,6 +148,22 @@
         <input class="st-input w-full" disabled name="activityType" value={span.type} />
       </Input>
 
+      {#if activityType && activityType.description}
+        <Input layout="inline">
+          <label use:tooltip={{ content: 'Activity Description', placement: 'top' }} for="activity-description">
+            Description
+          </label>
+          <textarea
+            class="st-input w-full"
+            disabled
+            name="activity-description"
+            id="activity-description"
+            value={activityType.description}
+            readonly
+          ></textarea>
```

Comment:
`<textarea>` can be self closed (i.e. `<textarea />` instead of <textarea></textarea>`)